xbmc/GUIInfoManager.cpp

```diff
@@ -6919,6 +6929,13 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     <p><hr>
 ///     @skinning_v21 **[New Infolabel]** \link ListItem_SongVideoURL `ListItem.SongVideoURL`\endlink
 ///   }
+///   \table_row3{   <b>`ListItem.BackendInstanceName`</b>,
+///                  \anchor ListItem_BackendInstanceName
+///                  _string_,
+///     @return The name used by the PVR client addon instance for the selected item.
+///     <p>hr>
+///     @skinning_v22 **[New Infolabel]** \link ListItem_BackendInstanceName `ListItem.BackendInstanceName`\endlink
```

Comment:
Place a `<p>` at next line, please. Look how it is done in existing documentaion.

Reply:
Done.